ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/PartyManagementServiceIT.scala

```diff
@@ -446,4 +449,91 @@ final class PartyManagementServiceIT extends PartyManagementITBase {
     }
   })
 
+  test(
+    "UpdateAnnotationsOfNonLocalParty",
+    "Update annotations of a non-local party",
+    enabled = _.userAndPartyLocalMetadataExtensions,
+    partyAllocation = allocate(SingleParty, NoParties),
+  )(implicit ec => { case Participants(Participant(alpha, alice), Participant(beta)) =>
+    val isDistinctServers = alpha.endpointId != beta.endpointId && alpha.ledgerId != beta.ledgerId
+    for {
+      alphaAliceO <- alpha.getParties(Seq(alice)).map(_.headOption)
+      betaAliceO <- beta.getParties(Seq(alice)).map(_.headOption)
+      _ = {
+        if (isDistinctServers) {
```

Comment:
How about moving this condition check even higher to reduce boilerplate as it looks like if `isDistinctServers == false` actually no other assertions are made?

could be something like
```
 case Participants(Participant(alpha, alice), Participant(beta)) if isDistinctServers => ...
 case Participants(Participant(alpha, alice), Participant(beta)) => Future.successful(()) //
```

Reply:
Wrapped the test body in a `if (isDistinctServers) { ... } else Future.sucessful(())`